src/routes/v1/friendships/history.rs

```diff
@@ -21,56 +21,35 @@ use crate::{
 
 use super::types::MessageRequestEvent;
 
-#[derive(Deserialize, Serialize)]
-pub struct RequestEventRequestBody {
-    pub timestamp_from: i64, // timestamp in milis
-    pub timestamp_to: i64,   // timestamp in milis
-}
-
 #[get("/v1/friendships/{friendshipId}/request-events/messages")]
 async fn get_sent_messages_request_event(
     req: HttpRequest,
-    body: web::Json<RequestEventRequestBody>,
     friendship_id: web::Path<Uuid>,
     app_data: Data<AppComponents>,
 ) -> Result<HttpResponse, FriendshipsError> {
-    let _logged_in_user: UserId = req
+    let logged_in_user: UserId = req
         .extensions()
         .get::<UserId>()
         .expect("to have a UserId")
         .clone();
 
-    // Convert it to a timestamp type that can be understood by PostgreSQL.
-    let timestamp_from_naive = match NaiveDateTime::from_timestamp_opt(body.timestamp_from, 0) {
-        Some(val) => val,
-        None => {
-            return Err(FriendshipsError::CommonError(CommonError::BadRequest(
-                "Failed to convert timestamp_to to NaiveDateTime".to_string(),
-            )))
-        }
+    // Retrieve users from friendship.
+    let users = match get_users_friendship(app_data.clone(), *friendship_id).await {
+        Some(value) => value,
+        None => return Err(FriendshipsError::CommonError(CommonError::NotFound)),
     };
 
-    let timestamp_to_naive = match NaiveDateTime::from_timestamp_opt(body.timestamp_to, 0) {
-        Some(val) => val,
-        None => {
-            return Err(FriendshipsError::CommonError(CommonError::BadRequest(
-                "Failed to convert timestamp_to to NaiveDateTime".to_string(),
-            )))
-        }
-    };
+    // Verify permissions.
```

Comment:
Let's add a more meaningful comment here, something like
```suggestion
    // Verify that the logged in user is present in the friendship
```